themes/V3/5ePHB/snippets/tableOfContents.gen.js

```diff
@@ -22,37 +22,39 @@ const getTOC = (pages)=>{
 		if(!_.last(_.last(res).children)) add2(null, page);
 		_.last(_.last(res).children).children.push({
 			title    : title,
-			page     : page + 1,
+			page     : page,
 			children : []
 		});
 	};
```

Comment:
We need functions for `add4` and `add5` here.

Reply:
I added h4, h5, and h6 here. I seem to have missed when in this timeline we added H6, but there you go.

The other PR has a replacement for this section of code that is recursive - though I need to test it still. Will resolve that out in #3254.